src/earn/EarnConfirmationScreen.tsx

```diff
@@ -33,12 +34,12 @@ import { getSerializablePreparedTransactions } from 'src/viem/preparedTransactio
 import { walletAddressSelector } from 'src/web3/selectors'
 import { isAddress } from 'viem'
 
-type Props = NativeStackScreenProps<StackParamList, Screens.EarnCollectScreen>
+type Props = NativeStackScreenProps<StackParamList, Screens.EarnConfirmationScreen>
 
-export default function EarnCollectScreen({ route }: Props) {
+export default function EarnConfirmationScreen({ route }: Props) {
   const { t } = useTranslation()
   const dispatch = useDispatch()
-  const { pool } = route.params
+  const { pool, mode, inputAmount, useMax } = route.params
```

Comment:
do we even need `useMax` as a parameter? Can't it just be inferred if inputAmount is undefined?

Reply:
I ran into the issue where the user starts a partial withdrawal, but then selects max; In this case, we want useMax to be true to ensure all the dust is withdrawn and the inputAmount !== undefined.